R/to_integer.R

```diff
@@ -210,6 +207,7 @@ to_integer.fixed_design <- function(x, sample_size = TRUE, ...) {
 #' @rdname to_integer
 #' @param round_up_final Events at final analysis is rounded up if `TRUE`;
 #' otherwise, just rounded, unless it is very close to an integer.
+#' @param ratio Positive integer for randomization ratio (experimental:control).
```

Comment:
Should we document what will happen if `ratio` is _not_ a positive integer or is this obvious?

```suggestion
#' @param ratio Positive integer for randomization ratio (experimental:control). A negative number will result in an error. A positive non-integer will result in `ratio = 0`.
```